Assets/BoundingsTimeout2D.cs

```diff
@@ -8,7 +8,6 @@ public class BoundingsTimeout2D : MonoBehaviour
 
     void Start()
     {
-        timeout = 3;
```

Comment:
For reference:
http://forum.unity3d.com/threads/who-wins-the-inspector-or-start-function.39018/